source/fundamentals/enterprise-auth.txt

```diff
@@ -93,9 +93,10 @@ mechanism:
       .. include:: /includes/fundamentals/code-snippets/auth-credentials-gssapi.rst
 
 
-In order to acquire a Kerberos ticket, the GSSAPI Java libraries require
-you to specify the realm and Key Distribution Center (KDC) system
-properties. See the sample settings in the example below:
+In order to acquire a 
+`Kerberos ticket <https://docs.oracle.com/javase/7/docs/api/javax/security/auth/kerberos/KerberosTicket.html>`__,
```

Comment:
Can we find a newer link?  We don't even support Java 7 anymore.   See if you can find something from Java 11, or at a minimum Java 8.

Reply:
Thanks for catching this Jeff. I wasn't thinking about different versions of Java when searching for this link, I will try to be sure to take into account Java SE version when linking to content going forward. Here is the fix:  https://github.com/mongodb/docs-java/pull/56/commits/0f68031b9e97c95ee27872eabcd961c1971afe07